CreatubblesAPIClient/Sources/Model/Creation.swift

```diff
@@ -189,6 +191,9 @@ open class Creation: NSObject, Identifiable {
 
         contentType = creationEntity.contentType ?? ""
 
+        let entityTags: Array<String> = creationEntity.tags.flatMap { $0.tag }
+        tags = entityTags.isEmpty ? nil : entityTags
```

Comment:
I thought about this, main reason why I left it in this way, was to distinguish situations:
"There were no tags set" vs "There was an empty list of tags" when parsing API